deploy/operator.yaml

```diff
@@ -20,7 +20,7 @@ spec:
     spec:
       containers:
       - name: portworx-operator
-        image: portworx/px-operator:1.10.0-dev
+        image: portworx/px-operator:1.10.1-dev
         imagePullPolicy: Always
         command:
         - /operator
```

Comment:
line 27, do we still need the `--verbose` here?

Reply:
any idea who uses this operator.yaml? the customer facing ones are spec gen and CSV which have been removed.

Reply:
I'm not sure but I think we can merge the change first.

Reply:
No one outside uses this. It is just for development.